src/zzz_od/api/utils/error_helpers.py

```
@@ -0,0 +1,223 @@
+    TemplateNotFoundError,
+    PermissionError
+)
+
+logger = logging.getLogger(__name__)
+
+F = TypeVar('F', bound=Callable[..., Any])
+
+
+def handle_battle_assistant_errors(func: F) -> F:
+    """
+    装饰器：自动处理战斗助手相关错误
```

Ruff (0.12.2)

Docstring contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF002)